jovo-platforms/jovo-platform-twilioautopilot/src/core/AutopilotResponse.ts

```diff
@@ -107,27 +107,14 @@ export class AutopilotResponse implements JovoResponse {
     return this.setSessionAttributes(sessionData);
   }
 
-  isTell(speechText?: string | string[]): boolean {
+  isTell(): boolean {
     const hasListenAction = this.actions.some((action) => {
       return action.listen;
     });
     // is ask()!
     if (hasListenAction) return false;
 
-    const sayAction = this.actions.find((action) => {
```

Comment:
We actually need that deleted part for the TestSuite, 
e.g. `expect(response.isTell('Hello World!')).toBe(true);`

But, we should check whether `speechText` is defined:

```ts
isTell(speechText?: string | string[]): boolean {
  const hasListenAction = this.actions.some((action) => {
    return action.listen;
  });
  // is ask()!
  if (hasListenAction) return false;

  const sayAction = this.actions.find((action) => {
    return action.say;
  });

  // no speech output in response
  if (!sayAction) return false;

  if (speechText) {
    if (Array.isArray(speechText)) {
      for (const speech of speechText) {
        if (AutopilotSpeechBuilder.toSSML(speech) === sayAction.say) return true;
      }
    }

    return AutopilotSpeechBuilder.toSSML(speechText as string) === sayAction.say;
  }

  return true;
}
```

That should fix the error that came up.